### PR TITLE
Fixes broken links and typos in managing automation content guide

### PR DESCRIPTION
**Pull request by hherbly**
This PR is associated with the JIRA tracking broken link fixes, [AAP-15176](https://issues.redhat.com/browse/AAP-15176).

### CODE REVIEW

File: downstream/modules/hub/proc-obtain-images.adoc [hunk 1]
```diff
@@ -58,6 +58,6 @@ $ podman images
 [role="_additional-resources"]
 .Additional resources
 
-* See link:redhat-connect.gitbook.io/catalog-help/[Red Hat Ecosystem Catalog Help] for information on registering and getting {ExecEnvShort}s.
+* See link:https://redhat-connect.gitbook.io/catalog-help/[Red Hat Ecosystem Catalog Help] for information on registering and getting {ExecEnvShort}s.
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-obtain-images.adoc:61]: Was removing the s a global change? If so, you missed this instance.

**Review by dcdacosta — APPROVED** [review]
One minor question. Otherwise, LGTM!

File: downstream/modules/hub/proc-obtain-images.adoc (hunk 1 above)
Comment (hherbly) [reply downstream/modules/hub/proc-obtain-images.adoc:61]: Good catch!

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**